test/spec/jdl/jdl_importer.spec.js

```diff
@@ -1865,5 +1865,60 @@ paginate * with infinite-scroll
         expect(fse.existsSync('.jhipster')).to.be.false;
       });
     });
+    context('when importing a JDL application with blueprints', () => {
+      let importState;
+
+      before(() => {
+        const importer = createImporterFromFiles([path.join('test', 'test_files', 'application_with_blueprints.jdl')], {
+          creationTimestamp: '2019-01-01'
+        });
+        importState = importer.import();
+      });
+      after(() => {
+        fse.removeSync('.yo-rc.json');
+      });
+
+      it('should return the blueprints attributes in the application', () => {
+        expect(importState.exportedApplications).to.deep.equal([
+          {
+            entities: [],
+            'generator-jhipster': {
+              applicationType: 'monolith',
+              authenticationType: 'jwt',
+              baseName: 'appWithBlueprints',
+              blueprints: ['generator-jhipster-vuejs', 'generator-jhipster-dotnetcore'],
```

Comment:
I think the generator doesn’t support simple arrays.
Should be `{name: blueprintName}`, version is not required.
https://github.com/jhipster/generator-jhipster/blob/10fa9ba215ede70096d880f804b1e4a0f1fd438b/test/blueprint/scoped-blueprint.spec.js#L63